sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -184,7 +187,195 @@ T fabs(T x);
 
 ===== Description
 
-Compute absolute value of a `bfloat16`.
+Compute absolute value of a `bfloat16` value or `sycl::marray<bfloat16, N>`.
+
+==== ceil
+
+```c++
+namespace sycl::ext::oneapi::experimental {
+template <typename T>
+T ceil(T x);
+} // namespace sycl::ext::oneapi::experimental
+```
+
+===== Description
+
+Round to integral value using the round to positive infinity rounding mode
```

Comment:
Nit, could be better to stick with the same pattern of descriptions as in the existing function definitions, e.g.:
```suggestion
Returns `x` rounded to an integral value using the round to positive infinity rounding mode
```